server/db/migrations/20190605110045-create-book.js

```diff
@@ -21,8 +21,13 @@ export function up(queryInterface, Sequelize) {
       type: Sequelize.NUMERIC
     },
     authorID: {
-      allowNull: true,
-      type: Sequelize.INTEGER
+      allowNull: false,
```

Comment:
I thought this was set to null for an reason

Reply:
I can change it back if it breaks anything. The reason I set it to not allow false is because then @PelumiAlesh has a validation check on it that states `"AuthorID can not be left empty"`